core/src/main/java/org/apache/calcite/adapter/enumerable/RexImpTable.java

```diff
@@ -216,6 +216,8 @@
 import static org.apache.calcite.sql.fun.SqlLibraryOperators.PARSE_URL;
 import static org.apache.calcite.sql.fun.SqlLibraryOperators.POW;
 import static org.apache.calcite.sql.fun.SqlLibraryOperators.REGEXP_CONTAINS;
```

Comment:
Is this PR for REGEXP_EXTRACT and REGEXP_EXTRACT_ALL? Can you update the title/commit message if so

Reply:
I know that's a bit confusing, this PR includes changes from EXTRACT cherrypicked here due to dependencies. Should be resolved when we merge that PR and rebase this one.